Scripts/Nuget.ps1

```diff
@@ -0,0 +1,47 @@
+# Specify the directory where you want to install NuGet
+$installPath = "$env:USERPROFILE\NuGet"
+$nugetPath = "$installPath\nuget.exe"
+
+# Exit the script if nuget.exe already exists, this assumes that nuget has been added to path
+if (Test-Path $nugetPath) {
+    Write-Host "Nuget already installed"
+    exit
+}
+
+# Create the directory if it doesn't exist
+if (-not (Test-Path $installPath)) {
+    New-Item -ItemType Directory -Path $installPath
+}
+
+# Download nuget.exe since it does not exist
+Invoke-WebRequest -Uri "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe" -OutFile $nugetPath
```

Comment:
Does the URL never change?  if no, we may want to update the repoConfiguration.json 
Do we have win-x64? if yes, let use the x64 version